TestCases/coupled_cht/disc_adj_unsteadyCHT_cylinder/chtMaster.cfg

```diff
@@ -0,0 +1,151 @@
+% 2021-03-11 TobiKattmann
+%
+SOLVER= MULTIPHYSICS
+%
+% Set RESTART_SOL=YES for primal runs including the FD sweep
+RESTART_SOL= NO
+RESTART_ITER= 2
+READ_BINARY_RESTART= YES
+SOLUTION_FILENAME= solution
+RESTART_FILENAME= solution
+%
+CONFIG_LIST = ( fluid.cfg, solid.cfg )
+%
+MARKER_ZONE_INTERFACE= ( cylinder_fluid, cylinder_solid )
+MARKER_CHT_INTERFACE=  ( cylinder_fluid, cylinder_solid )
+%
+% ------------------------- UNSTEADY SIMULATION -------------------------------%
+%
+TIME_DOMAIN= YES
+TIME_MARCHING= DUAL_TIME_STEPPING-2ND_ORDER
+%
+TIME_STEP= 500
+%
+MAX_TIME= 1e9
+% For a primal restart change TIME_ITER=56 for the correct number of steps. 54 is for the adjoint run.
+TIME_ITER= 54
+% For the primal (and therefore FD sweep) OUTER_ITER=200 is suitable.
+% For an accurate adjont run set OUTER_ITER=500. 100 is for the regression test.
+OUTER_ITER= 100
+%INNER_ITER= 1
+%
+UNST_ADJOINT_ITER= 56
+%
+ITER_AVERAGE_OBJ= 54
+%
+% ------------------------- INPUT/OUTPUT FILE INFORMATION --------------------------%
+%
+MESH_FILENAME= MeshCHT.su2
+%
+SCREEN_OUTPUT= (TIME_ITER, OUTER_ITER, BGS_ADJ_PRESSURE[0], BGS_ADJ_VELOCITY-X[0], BGS_ADJ_VELOCITY-Y[0], BGS_ADJ_TEMPERATURE[0], BGS_ADJ_TEMPERATURE[1] )
```

Comment:
It would be good to throw a sensitivity output in here (the geometric and AoA perhaps) for extra coverage since those can break independent of the adjoints.

Reply:
Yeah, I tested 2 or 3 other things and they had 0.0 output so I ditched that. But I'll look again what nice output there is and add that (tomorrow)

Reply:
Done